src/views/simulation/resultats.vue

```diff
@@ -50,7 +50,7 @@
         >
         Les montants avancés sont arrondis à une dizaine d'euros près :
       </p>
-      <BenefitsList :benefits="benefits" />
+      <BenefitsList :benefits-or-benefit-groups="benefitTree" />
```

Comment:
Ah désolé, je n'ai pas explicité certaines de mes propositions.

Ici il s'agit bien de `benefits-and-benefits-groups` c'est quand je manipule un élément de cette liste que j'ai utilisé `benefitOrBenefitGroup`. 

Pour moi, le and assume qu'on mélange des  `benefits` et de `benefitGroups` alors que le or dans ce contexte pourrait signifier qu'on a soit une liste de `benefits` soit une liste de `benefitGroups`.